coordinator/points_writer.go

```diff
@@ -312,14 +310,18 @@ func (w *PointsWriter) WritePoints(database, retentionPolicy string, consistency
 
 	// Send points to subscriptions if possible.
 	ok := false
+	pts := &WritePointsRequest{Database: database, RetentionPolicy: retentionPolicy, Points: points}
 	// We need to lock just in case the channel is about to be nil'ed
 	w.mu.RLock()
-	select {
-	case w.subPoints <- &WritePointsRequest{Database: database, RetentionPolicy: retentionPolicy, Points: points}:
-		ok = true
-	default:
+	for _, ch := range w.subPoints {
+		select {
+		case ch <- pts:
+			ok = true
+		default:
+		}
 	}
 	w.mu.RUnlock()
+
 	if ok {
 		atomic.AddInt64(&w.stats.SubWriteOK, 1)
```

Comment:
This stat now counts if at least one succeed. Do we want to change to count if all succeeded instead of at least one succeeds?